### PR TITLE
Improves coverage of user tests and model methods

### PR DESCRIPTION
**Pull request by helrond**
Adds coverage for user views and model methods. Adds a `name` attribute to the `SplashView` for easy reference.

### CODE REVIEW

File: aurora/bag_transfer/test/test_users.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+from django.urls import reverse
+
+
+class UserTestCase(TestCase):
+    fixtures = ["complete.json"]
+
+    def setUp(self):
+        self.client.force_login(User.objects.get(username="admin"))
+
+    def assert_status_code(self, method, url, data, status_code):
```
Comment (bonniegee) [inline aurora/bag_transfer/test/test_users.py:15]: This is more of a maybe-add-this-as-an-issue thing, but could this be added to `helpers.py` and incorporated into the other tests?

**Review by bonniegee — APPROVED** [review]

File: aurora/bag_transfer/test/test_users.py (hunk 1 above)
Comment (helrond) [reply aurora/bag_transfer/test/test_users.py:15]: @bonniegee yes absolutely - I have a not-yet-filed issue about that. I was waiting to see if other things surfaced while re-writing tests.

**Review by HaSistrunk — APPROVED** [review]

**Review by p-galligan — APPROVED** [review]
I got a couple of django warnings when running tests:

`WARNING:django.request:Not Found: /app/rights/65/delete/`
`WARNING:django.request:Not Found: /app/orgs/2/bagit_profile/delete/`

The tests pass though, so I'm good. Just wanted to point them out.

**Comment by helrond** [comment]
Yup, I'm getting those too. I'm going to do a more holistic scan of unit tests as part of #475 once all these PRs are merged.

**Pull request merged**